docs/pages/build-reference/apk.md

```diff
@@ -4,11 +4,17 @@ title: Building APKs for Android emulators and devices
 
 The default file format used when building Android apps with EAS Build is an [Android App Bundle](https://developer.android.com/platform/technology/app-bundle) (AAB / **.aab**). This format is optimized for distributing to the Google Play Store, but AABs can't be installed directly to your device. To install a build directly to your Android device or emulator, you need to build an [Android Package](https://en.wikipedia.org/wiki/Android_application_package) (APK / **.apk**) instead.
 
+## Prerequisites
```

Comment:
i don't think it's necessary to add this. we would need to include this on most pages in the eas build docs if we wanted them to stand completely on their own, along with probably a bunch of other redundant information. it is assumed if you are using this page that you already have eas build running. that said, i'm open to ideas for how to hint to people that they should refer back to https://docs.expo.dev/build/introduction/ if they haven't used eas build before and land on this page